internal/commands/image/rm.go

```diff
@@ -0,0 +1,56 @@
+package image
+
+import (
+	"fmt"
+
+	"github.com/docker/app/internal/store"
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/config"
+	"github.com/docker/distribution/reference"
+	"github.com/spf13/cobra"
+)
+
+func rmCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "rm [APP_IMAGE] [APP_IMAGE...]",
+		Short: "Remove an application image",
+		Args:  cli.RequiresMinArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			appstore, err := store.NewApplicationStore(config.Dir())
+			if err != nil {
+				return err
+			}
+
+			bundleStore, err := appstore.BundleStore()
+			if err != nil {
+				return err
+			}
+
+			for _, arg := range args {
+				ref, err := reference.ParseNormalizedNamed(arg)
+				if err != nil {
+					fmt.Println("Error:", err.Error())
+					continue
+				}
+
+				rmErr := runRm(bundleStore, arg)
+				if rmErr != nil {
```

Comment:
Another way would be to stack the errors into a slice of errors and concatenate everything into a single error and return it?

Reply:
We don't really want to return the errors, we only want to know if one of the removes had an error so that we can exit with the right exit code.

Reply:
I think we should let cobra print the error(s) and return the exit code. That's why I suggest we should return the errors here.

Reply:
> Another way would be to stack the errors into a slice of errors and concatenate everything into a single error and return it?

https://github.com/hashicorp/go-multierror is nice to do that.

In short:

```go
if err := step1(); err != nil {
	result = multierror.Append(result, err)
}
if err := step2(); err != nil {
	result = multierror.Append(result, err)
}

return result.ErrorOrNil()
```

Reply:
I want a nice output (the same as for `docker image rm`):

```bash
$ docker app image rm my-app:1.0.0 my-app:2.0.0 my-app:1.0.1
Deleted: docker.io/library/my-app:1.0.0
Error: no such image docker.io/library/my-app:2.0.0
Deleted: docker.io/library/my-app:1.0.1
```

That's why I can't let cobra or multierror do the output.